Tools/WinMLRunner/src/Run.cpp

```diff
@@ -392,65 +448,21 @@ HRESULT EvaluateModels(const std::vector<std::wstring>& modelPaths, const std::v
                        << std::endl;
             continue;
         }
-
         for (const auto& deviceType : deviceTypes)
         {
-            for (const auto& inputBindingType : inputBindingTypes)
+            HRESULT evaluateModelGivenDeviceTypeResult =
+                EvaluateModelGivenDeviceType(model, deviceType, inputBindingTypes, inputDataTypes,
```

Comment:
Can we rename this to EvaluateModelWithDeviceType?

Reply:
Sure sure! I can rename this.